.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,18 @@
+name: Feature request
```

Comment:
Nit, suggest to use `Feature Request` to be consistent with other templates (e..g. `Bug Report`, `Flaking Test`)

Reply:
Could you provide more details about what parts of templates are not consistent?

Reply:
It's nit as I said in how the name is mentioned :) : 
`name: Bug Report` in the bug-report.yml
`name: Flaking Test` in the test-flake.yml 
But here in the feature-request.yml it's `name: Feature request`   (i.e. `request` vs `Request`)

Reply:
If you are making other changes, good to update here too.